extension/lib/settings_view.js

```diff
@@ -118,66 +131,6 @@ class SettingsView {
         }
     }
 
-    // Update styles of customization buttons so that the one that is lighted up switches
-    // when the user clicks on another one
-    updateCustomizationButtonStyles(setting, choice) {
-        let current_keyword_setting = this.keywordSetting;
-        let current_highlighter_setting = this.highlighterSetting;
-        let current_shadow_setting = this.shadowSetting;
-        switch(setting) {
-            case "keyword":
-                switch(choice) {
-                    case "Green":
-                        // Turn new one to high opacity, old one to low opacity
-                        document.getElementById("customKeywordGreen").style.fillOpacity = "1";
```

Comment:
i think this one is one of the bug causes.
you should turn off first, then turn on.
Otherwise, for the same element, you will effectively turn off.